go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ultravioletrs/cocos
 
-go 1.23.0
+go 1.22.5
 
```

Comment:
What's the issue with buildroot that requires us to downgrade go version?

Reply:
https://github.com/buildroot/buildroot/tree/master/package/go

Reply:
Why is this downgraded?

Reply:
buildroot does not support go 1.23 yet